docs/supported-components.human

```diff
@@ -46,7 +46,7 @@ memcached
 mongodb-core
   2.0.0-3.2.7
 mongodb
-  3.3.0-3.7.0
+  3.3.0-3.7.1
```

Comment:
mongoose is missing because it timeout out at `5.12.7` thus no `.json` produced.
 The real data is:
`4.6.4-4.10.8`
`4.11.1`
`4.11.3-5.13.9`